cluster/images/hyperkube/master-multi.json

```diff
@@ -40,6 +61,25 @@
               "--master=127.0.0.1:8080",
               "--v=2"
         ]
+    },
+    {
+      "name": "setup",
+      "image": "gcr.io/google_containers/hyperkube-ARCH:VERSION",
```

Comment:
Why do you change the repo name? It's supposed to be `gcr.io/google_containers/hyperkube:VERSION`.